pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>com.expediagroup</groupId>
     <artifactId>eg-oss-parent</artifactId>
-    <version>1.1.0</version>
+    <version>1.2.0-SNAPSHOT</version>
```

Comment:
This will obviously go to `1.2.0` when the parent is released and before this is merged.